HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -1688,34 +1710,50 @@
+      <sch:assert role='ERROR' test='count(h:PerformanceDataPoint[h:OutdoorTemperature &gt; 95 and h:CapacityDescription="nominal"]) &lt;= 1'>Expected 0 or 1 element(s) for xpath: PerformanceDataPoint[OutdoorTemperature&gt;95 and CapacityDescription="nominal"]</sch:assert> <!-- See [PerformanceDataPoint] -->
+      <sch:assert role='ERROR' test='count(h:PerformanceDataPoint[h:OutdoorTemperature &gt; 95 and h:CapacityDescription="maximum"]) &lt;= 1'>Expected 0 or 1 element(s) for xpath: PerformanceDataPoint[OutdoorTemperature&gt;95 and CapacityDescription="maximum"]</sch:assert> <!-- See [PerformanceDataPoint] -->
       <sch:assert role='ERROR' test='count(h:PerformanceDataPoint[h:OutdoorTemperature=95 and h:CapacityDescription="minimum"]) = 1'>Expected 1 element(s) for xpath: PerformanceDataPoint[OutdoorTemperature=95 and CapacityDescription="minimum"]</sch:assert> <!-- See [PerformanceDataPoint] -->
+      <sch:assert role='ERROR' test='count(h:PerformanceDataPoint[h:OutdoorTemperature=95 and h:CapacityDescription="nominal"]) = 1'>Expected 1 element(s) for xpath: PerformanceDataPoint[OutdoorTemperature=95 and CapacityDescription="nominal"]</sch:assert> <!-- See [PerformanceDataPoint] -->
       <sch:assert role='ERROR' test='count(h:PerformanceDataPoint[h:OutdoorTemperature=95 and h:CapacityDescription="maximum"]) = 1'>Expected 1 element(s) for xpath: PerformanceDataPoint[OutdoorTemperature=95 and CapacityDescription="maximum"]</sch:assert> <!-- See [PerformanceDataPoint] -->
-      <sch:assert role='ERROR' test='count(h:PerformanceDataPoint[h:OutdoorTemperature!=95 and h:CapacityDescription="minimum"]) &gt;= 1'>Expected 1 or more element(s) for xpath: PerformanceDataPoint[OutdoorTemperature!=95 and CapacityDescription="minimum"]</sch:assert> <!-- See [PerformanceDataPoint] -->
-      <sch:assert role='ERROR' test='count(h:PerformanceDataPoint[h:OutdoorTemperature!=95 and h:CapacityDescription="maximum"]) &gt;= 1'>Expected 1 or more element(s) for xpath: PerformanceDataPoint[OutdoorTemperature!=95 and CapacityDescription="maximum"]</sch:assert> <!-- See [PerformanceDataPoint] -->
+      <sch:assert role='ERROR' test='count(h:PerformanceDataPoint[h:OutdoorTemperature=82 and h:CapacityDescription="minimum"]) = 1'>Expected 1 element(s) for xpath: PerformanceDataPoint[OutdoorTemperature=82 and CapacityDescription="minimum"]</sch:assert> <!-- See [PerformanceDataPoint] -->
+      <sch:assert role='ERROR' test='count(h:PerformanceDataPoint[h:OutdoorTemperature=82 and h:CapacityDescription="nominal"]) = 1'>Expected 1 element(s) for xpath: PerformanceDataPoint[OutdoorTemperature=82 and CapacityDescription="nominal"]</sch:assert> <!-- See [PerformanceDataPoint] -->
```

Comment:
There is no nominal/rated 82F datapoint, we can't require this. Let's plan to discuss. (If we update this, also need to update docs and changelog.)

Reply:
I found that issue today as well. Currently made up some numbers there but we need to handle these datapoints as optional inputs

Reply:
The 5F nominal/rated datapoint should also be optional.